dashboard/app/views/teacher_feedbacks/index.html.haml

```diff
@@ -1,5 +1,5 @@
 - feedback_data = {}
-- feedback_data[:all_feedback] = @feedbacks_as_student_with_level_info
+- feedback_data[:all_feedback_by_level] = @feedbacks_by_level
```

Comment:
Do we want to use `feedback` or `feedbacks` as the plural form?  (I don't have a strong opinion of which we use as long as we are consistent.)